src/test/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        //excavator.setName("excavator");
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after object instantiation.